src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -54,14 +53,14 @@ const char *program_name;
 
 TDB_CONTEXT *db = nullptr;
 
-#define KEY_LAST_ACTIVITY            "last-activity"
-#define KEY_PERIOD_START             "period-start"
-#define KEY_PERIOD_LENGTH_CONFIGURED "period-length-configured"
-#define KEY_TIME_BUDGET_LEFT         "time-budget-left"
-#define KEY_TIME_BUDGET_CONFIGURED   "time-budget-configured"
+static const auto KeyLastActivity = "last-activity";
+static const auto KeyPeriodStart = "period-start";
+static const auto KeyPeriodLengthConfigured = "period-length-configured";
+static const auto KeyTimeBudgetLeft = "time-budget-left";
+static const auto KeyTimeBudgetConfigured = "time-budget-configured";
 
 /** Maximum size of buffers used to read or display lines. */
-#define TQ_BUFFERSIZE                     1024
+const size_t TQ_BUFFERSIZE = 1024;
```

Comment:
If it is still used:

```suggestion
static const size_t TQ_BUFFERSIZE = 1024;
```

otherwise:

```suggestion
```

Reply:
it is; making static